provisioner.go

```diff
@@ -653,5 +662,10 @@ func canonicalizeConfig(data *ConfigData) (cfg *Config, err error) {
 	} else {
 		cfg.CmdTimeoutSeconds = defaultCmdTimeoutSeconds
 	}
+	if data.FolderExpression != "" {
+		cfg.FolderExpression = data.FolderExpression
```

Comment:
I'm thinking of we can validate the input of the `data.FolderExpression`?
If there is a typo or capitalization prob, for example, `data.FolderExpression = {{.PVName}}-{{.namespace}}-{{.pvcNane}}`, what will happen?